src/adapters/adapter.js

```diff
@@ -191,8 +191,7 @@ class Adapter {
       case 206:
```

Comment:
Now that we handle 206, I suppose we should remove this error handling here? Sorry I missed it in the earlier review.

Reply:
My bad. Done.